pkg/collector/runner/runner_test.go

```diff
@@ -24,6 +24,10 @@ import (
 // Fixtures
 
 type testCheck struct {
+	// We have to align the atomics to 64-bit boundaries so these are put down first
+	runCount uint64
+	stopped  uint64
```

Comment:
Per discussion, it looks like this is a [bug in staticcheck](https://staticcheck.io/docs/checks#SA1027), since (a) this wasn't a 32-bit access and (b) it was 64-bit aligned.. but fixing it by moving fields is easy.